### PR TITLE
if inline style was applied by other plugin md plugin should not unst…

### PR DESCRIPTION
**Pull request by andrewfan**
…ick it

### CODE REVIEW

File: src/__test__/plugin.test.js [hunk 1]
```diff
@@ -13,6 +13,11 @@ import {
 import { Map, List } from "immutable";
 import createMarkdownPlugin from "../";
 
+const applyMDtoInlineStyleChange = editorState =>
```
Comment (andrewfan) [inline src/__test__/plugin.test.js]: not sure what is better
- copy this function from new tests and leave it here until old styled tests will be migrated (will they?)
- simply import it from new tests

**Comment by andrewfan** [comment]
fixes for https://github.com/withspectrum/draft-js-markdown-plugin/issues/122

File: src/__test__/plugin.test.js (hunk 1 above)
Comment (mxstbr) [reply src/__test__/plugin.test.js]: Simply import it from new tests, old tests will stay as they are I'm pretty sure!

**Review by mxstbr — APPROVED** [review]
I think it looks good?

**Pull request merged**